src/lib.rs

```diff
@@ -143,6 +143,8 @@ pub mod adc;
 #[cfg(riot_module_periph_dac)]
 pub mod dac;
 
```

Comment:
How do I make `#[cfg(.._periph_uart)]` work? (rustc complaints that config doesn't exist)

Reply:
There is an allow-list of known modules in build.rs